src/apps/relay/dbdrivers/dbd_redis.c

```diff
@@ -134,9 +134,9 @@ static Ryconninfo *RyconninfoParse(const char *userdb, char **errmsg) {
       } else if (!strcmp(s, "secret")) {
         co->password = strdup(seq + 1);
       } else if (!strcmp(s, "port")) {
-        co->port = (unsigned int)atoi(seq + 1);
+        co->port = atoi(seq + 1);
```

Comment:
don't you need a cast to uint16_t to avoid the compiler warning about precision loss?